aten/src/ATen/native/native_functions.yaml

```diff
@@ -5969,6 +5969,13 @@
   dispatch:
     CUDA: _amp_non_finite_check_and_unscale_cuda_
```

Comment:
do we still need _amp_non_finite_check_and_unscale?

Reply:
Good point, we don't need it to be Python-exposed anymore.